aten/src/ATen/native/cudnn/ConvPlaceholders.cpp

```diff
@@ -150,6 +150,14 @@ Tensor cudnn_convolution_deprecated2(
   return at::cudnn_convolution(input_t, weight_t, padding, stride, dilation, groups, benchmark, deterministic, at::globalContext().allowTF32CuDNN());
 }
 
+//Tensor cudnn_convolution_deprecated3(
+//    const Tensor& input_t, const Tensor& weight_t,
+//    IntArrayRef padding, IntArrayRef stride, IntArrayRef dilation,
+//    int64_t groups, bool benchmark)
+//{
+//  return at::cudnn_convolution(input_t, weight_t, padding, stride, dilation, groups, benchmark, true, at::globalContext().allowTF32CuDNN());
+//}
+
```

Comment:
just a dummy function to uncomment after uncommenting the corresponding decl in `native_functions.yaml`